ts/features/itwallet/common/utils/itwCredentialIssuanceUtils.ts

```diff
@@ -120,18 +77,18 @@ export const obtainCredential = async ({
   credentialType,
   requestedCredential,
   pid,
-  wiaCryptoContext,
   walletInstanceAttestation,
   clientId,
   codeVerifier,
   credentialDefinition,
   issuerConf
 }: ObtainCredentialParams) => {
-  await regenerateCryptoKey(DPOP_CREDENTIAL_KEYTAG);
+  // Get WIA crypto context
+  const wiaCryptoContext = createCryptoContextFor(WIA_KEYTAG);
 
   // Create PID and DPoP crypto context;
   const pidCryptoContext = createCryptoContextFor(pid.keyTag);
-  const dPopCryptoContext = createCryptoContextFor(DPOP_CREDENTIAL_KEYTAG);
+  const dPopCryptoContext = createCryptoContextFor(DPOP_KEYTAG);
```

Comment:
I am getting an error because the public key associated to that key tag does not exist. When do we regenerate the `DPOP_KEYTAG`?

Reply:
`DPOP_KEYTAG` is regenerated during the identification flow. I have added the generation also in this flow here: https://github.com/pagopa/io-app/pull/6225/commits/2428357681677f5c95ddd20debe6e3ef359a2623

Reply:
I already had a valid eID, so I didn't need to go through identification again. The key tag was the older one, so that was probably the issue.

I think it is safe to regenerate it anyway to cover this case.

Reply:
The `DPOP_KEYTAG` **must** be regenerated for each issuance flow but it can be reused within the same flow